json.go

```diff
@@ -63,3 +63,27 @@ func (this *JSONFile) flatten(inputJSON map[string]interface{}, namespace string
 
 	return flattened, nil
 }
+
+//FlatJSON convert the input into map[string]string and adds namespace.
+func FlatJSON(input map[string]interface{}, namespace string) (map[string]string, error) {
```

Comment:
Instead of separating this function out, I'd suggest inheriting from the `JSONFile` struct. Check out the pattern used for the `YAMLFile` provider